pvm.py

```diff
@@ -36,29 +37,90 @@ def _init_logger():
 VIDEO_PATH = "jellyfish720p.mp4"
 media = ""
 IS_FILE_SET = False
+NEXT_TIME = None
+
+'''
+UDP command example:
+/PVM 12:13:14
+/PVM <action_name> <action_value>
```

Comment:
```suggestion
/PVM HH MM SS <action_name> <action_value>
```
I could also suggest changing `<action_name>` to `<action_command>` or simply making it `<command> <value>`

Reply:
Done